03_Data_Management/Sampling_toolset.Rmd

```diff
@@ -1,36 +0,0 @@
-
```

Comment:
why did you delete this file?

Reply:
@ratnanil  I did not delete it. I renamed is as to Sampling.Rmd. I did that to keep some consistency. Although all Rmd file represent a toolset, only some of them included the _toolset in their naming. It's just a proposal. We can discuss it